packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/__tests__/quanticGeneratedAnswer.test.js

```diff
@@ -724,15 +742,50 @@ describe('c-quantic-generated-answer', () => {
         generatedAnswerState = initialGeneratedAnswerState;
       });
 
-      it('should not display the generated answer card', async () => {
-        const element = createTestComponent();
-        await flushPromises();
+      describe('when no custom error message slot is provided', () => {
+        it('should not display the generated answer card', async () => {
+          const element = createTestComponent();
+          await flushPromises();
 
-        const generatedAnswerCard = element.shadowRoot.querySelector(
-          selectors.generatedAnswerCard
-        );
+          const generatedAnswerCard = element.shadowRoot.querySelector(
+            selectors.generatedAnswerCard
+          );
+          expect(generatedAnswerCard).toBeNull();
+
+          const generatedAnswer = element.shadowRoot.querySelector(
+            selectors.generatedAnswer
+          );
+          expect(generatedAnswer).toBeNull();
+        });
+      });
+
+      describe('when a custom error message slot is provided', () => {
+        beforeEach(() => {
+          generatedAnswerState = {
+            ...initialGeneratedAnswerState,
+            cannotAnswer: true,
+          };
+          mockSuccessfulHeadlessInitialization();
+          prepareHeadlessState();
+        });
+
+        it('should properly display the no generated answer template', async () => {
+          const element = createTestComponent(
+            {...defaultOptions},
```

Comment:
no need for destructuring when just passing the object.
```suggestion
            defaultOptions,
```